script.js

```diff
@@ -0,0 +1,32 @@
+function Calculator() {
+  this.operandContainer = []
+  this.operatorContainer = []
+}
+
+Calculator.prototype.evaluate = function evaluate() {
+  // do the function logic
+}
+
+Calculator.prototype.sendOperandToContainer = function sendOperand( number ) {
+  //is operand stack empty?
+    //yes: push operand
+    //no: was last thing pushed into a stack an operator?
+      // yes: push operand
+      // no: concat number
+}
+
+Calculator.prototype.sendOperatorToContainer = function sendOperatorToContainer( operator ) {
+  //is operator stack empty
```

Comment:
I don't think you need to repeat the name like this.  Just do `Calculator.prototype.sendOperatorToContainer = function ( operator ) {`
But I'd test it to make sure.